extensions/arc/src/ui/dashboards/postgres/postgresExtensionsPage.ts

```diff
@@ -0,0 +1,164 @@
+	}
+
+	protected get title(): string {
+		return loc.preLoadedExtensions;
+	}
+
+	protected get id(): string {
+		return 'postgres-extensions';
+	}
+
+	protected get icon(): { dark: string; light: string; } {
+		return IconPathHelper.extensions;
+	}
+
+	protected get container(): azdata.Component {
+		const root = this.modelView.modelBuilder.divContainer().component();
+		const content = this.modelView.modelBuilder.divContainer().component();
+		root.addItem(content, { CSSStyles: { 'margin': '10px 20px 0px 20px' } });
+
+		content.addItem(this.modelView.modelBuilder.text().withProps({
+			value: this.title,
+			CSSStyles: { ...cssStyles.title }
+		}).component());
+
+		content.addItem(this.modelView.modelBuilder.text().withProps({
+			value: loc.extensionsDescription,
+			CSSStyles: { ...cssStyles.text, 'margin-block-start': '0px', 'margin-block-end': '0px' }
+		}).component());
+
+		const info = this.modelView.modelBuilder.text().withProperties<azdata.TextComponentProperties>({
+			value: loc.extensionsFunction,
+			CSSStyles: { ...cssStyles.text, 'margin-block-start': '0px', 'margin-block-end': '0px' }
+		}).component();
+
+		const link = this.modelView.modelBuilder.hyperlink().withProperties<azdata.HyperlinkComponentProperties>({
+			label: loc.extensionsLearnMore,
+			url: 'https://docs.microsoft.com/azure/azure-arc/data/get-connection-endpoints-and-connection-strings-postgres-hyperscale',
+		}).component();
+
+		const infoAndLink = this.modelView.modelBuilder.flexContainer().withLayout({ flexWrap: 'wrap' }).component();
+		infoAndLink.addItem(info, { CSSStyles: { 'margin-right': '5px' } });
+		infoAndLink.addItem(link);
+		content.addItem(infoAndLink, { CSSStyles: { 'margin-bottom': '15px', 'margin-top': '25px' } });
+
+		this.extensionsTable = this.modelView.modelBuilder.declarativeTable().withProperties<azdata.DeclarativeTableProperties>({
+			ariaLabel: loc.extensionsTableLabel,
+			width: '100%',
+			columns: [
+				{
+					displayName: loc.extensionName,
+					valueType: azdata.DeclarativeDataType.string,
+					isReadOnly: true,
+					width: '100%',
+					headerCssStyles: cssStyles.tableHeader,
+					rowCssStyles: cssStyles.tableRow
+				}
+			],
+			data: []
+		}).component();
+
+		this.extensionsLoading = this.modelView.modelBuilder.loadingComponent()
+			.withItem(this.extensionsTable)
+			.withProperties<azdata.LoadingComponentProperties>({
+				loading: !this._postgresModel.configLastUpdated,
+				loadingText: loc.extensionsTableLoading,
+				loadingCompletedText: loc.extensionsTableLoadingComplete
+			}).component();
+
+		content.addItem(this.extensionsLoading, { CSSStyles: cssStyles.text });
+
+		this.initialized = true;
+		return root;
+	}
+
+	protected get toolbarContainer(): azdata.ToolbarContainer {
+		// Add extensions
+		const addExtensionsButton = this.modelView.modelBuilder.button().withProperties<azdata.ButtonProperties>({
```

Comment:
If there's a way to add extensions, there should also be a way to remove extensions.  Don't know exactly what it would look like.  Perhaps each row of the table could have a delete button that removes that particular extension.

Reply:
I am separating the work of adding/dropping extensions to another PR. It will be with having a delete button column.